.travis.yml

```diff
@@ -35,6 +35,10 @@ before_deploy:
   - awk -F" = " -v updatedVal="= $GMAIL_APP_PASSWORD" '/sendPassword =/{$2=updatedVal}1' input.properties > on.properties
   - awk -F" = " -v updatedVal="= true" '/shouldSendEmails =/{$2=updatedVal}1' on.properties > emailer.properties
   - rm input.properties on.properties
+  - cp slack.properties.example input.properties
```

Comment:
I was thinking about that too--I think we should at some point.
I was considering moving it to a python script for readability but it doesn't look like it would be any easier in python.